src/validator/GoValidator.cpp

```diff
@@ -683,6 +684,25 @@ std::vector<storage::cpp2::VertexProp> GoValidator::buildDstVertexProps() {
     return vertexProps;
 }
 
+std::vector<std::string> GoValidator::buildDstVertexColNames() {
+    std::vector<std::string> dstColNames;
+    dstColNames.reserve(exprProps_.dstTagProps().size());
+    dstColNames.emplace_back(kVid);
+    for (const auto &tagProps : exprProps_.dstTagProps()) {
+        auto tagId = tagProps.first;
+        auto tagNameResult = qctx_->schemaMng()->toTagName(space_.id, tagId);
+        DCHECK(tagNameResult.ok());
+        if (!tagNameResult.ok()) {
```

Comment:
if `tagNameResult` is not ok, couldn't make validator continue. I think the tagName can be saved in exprProps_, so here not need to get tagName from tagId.

Reply:
This had be checked out of this functions, for more independent we could return error.

Reply:
But I don't think it's meanful.

Reply:
Fixed.